qutebrowser/browser/commands.py

```diff
@@ -277,16 +277,9 @@ def tab_pin(self, count=None):
         """
         tab = self._cntwidget(count)
         if tab is None:
-            return
+            raise cmdexc.CommandError("Tab {} does not exist!".format(count))
```

Comment:
Ah, I never actually noticed that :P.

Yup, this error does seem out of place, I'll remove it :).